From f9ce07e04e02166727d547a7f28c93d475d43ed5 Mon Sep 17 00:00:00 2001
From: Timo Aaltonen <tjaalton@debian.org>
Date: Wed, 17 Dec 2014 14:31:59 +0200
Subject: [PATCH 2/2] Revert "drm/i915: send proper opregion notifications on
 suspend/resume"

This reverts commit e5747e3adcd67ae27105003ec99fb58cba180105.

Conflicts:
	drivers/gpu/drm/i915/i915_drv.c
---
 drivers/gpu/drm/i915/i915_drv.c | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_drv.c b/drivers/gpu/drm/i915/i915_drv.c
index 2c44369..715bd9f 100644
--- a/drivers/gpu/drm/i915/i915_drv.c
+++ b/drivers/gpu/drm/i915/i915_drv.c
@@ -28,7 +28,6 @@
  */
 
 #include <linux/device.h>
-#include <linux/acpi.h>
 #include <drm/drmP.h>
 #include <drm/i915_drm.h>
 #include "i915_drv.h"
@@ -558,7 +557,6 @@ static int i915_drm_suspend(struct drm_device *dev)
 {
 	struct drm_i915_private *dev_priv = dev->dev_private;
 	struct drm_crtc *crtc;
-	pci_power_t opregion_target_state;
 
 	/* ignore lid events during suspend */
 	mutex_lock(&dev_priv->modeset_restore_lock);
@@ -609,12 +607,6 @@ static int i915_drm_suspend(struct drm_device *dev)
 
 	i915_save_state(dev);
 
-	if (acpi_target_system_state() >= ACPI_STATE_S3)
-		opregion_target_state = PCI_D3cold;
-	else
-		opregion_target_state = PCI_D1;
-	intel_opregion_notify_adapter(dev, opregion_target_state);
-
 	intel_uncore_forcewake_reset(dev, false);
 	intel_opregion_fini(dev);
 
@@ -730,8 +722,6 @@ static int i915_drm_resume(struct drm_device *dev)
 	dev_priv->modeset_restore = MODESET_DONE;
 	mutex_unlock(&dev_priv->modeset_restore_lock);
 
-	intel_opregion_notify_adapter(dev, PCI_D0);
-
 	drm_kms_helper_poll_enable(dev);
 
 	return 0;
-- 
2.1.3

